wirecard/oxid-ee
# WDPD-65 Log user actions

### This PR

* Adds information if the transaction was possibly manipulated
* Adds missing translations
* Adds missing copyright text in various files

### How to Test

* Make an order on frontend.
* In admin panel click on Administer Orders -> Orders and then click on the newly created order.
* In order overview tab panel, click on "Transactions".
* You will see additional column "Secured" in transaction table. The value can be "secured" or "manipulated".

### Jira Links

* https://jira.parkside.at/browse/WDPD-65
